clients/src/main/java/org/apache/kafka/clients/consumer/internals/CoordinatorRequestManager.java

```diff
@@ -122,6 +123,23 @@ NetworkClientDelegate.UnsentRequest makeFindCoordinatorRequest(final long curren
         });
     }
 
+    /**
+     * Handles the disconnection of the current coordinator.
+     * This method checks if the given exception is an instance of {@link DisconnectException}.
+     * If so, it marks the coordinator as unknown, indicating that the client should
+     * attempt to discover a new coordinator. For any other exception type, no action is performed.
+     *
+     * @param exception     The exception to handle, which was received as part of a request response.
+     *                      If this is an instance of {@link DisconnectException}, the coordinator is marked as unknown.
+     *                      For other types of exceptions, no action is performed.
```

Comment:
nit: this is already said right above, so let's just remove it from here?

Reply:
Yes, it's a little verbose. I will remove it 👍🏼